### ISSUE TITLE
Antialiasing broken in IE11?

### ISSUE THREAD

**Issue by fredludlow** [arose/ngl#103]
The latest version on the demo page (and my local copy which is at arose@fbf022d)

seem to be broken on IE 11 with the following console error:
WEBGL11049: INVALID_ENUM: texImage2D: Invalid type

Loading an example initially renders but on rotation dissapears (with a string of errors in the console along the lines of 
drawArrays: The texture is a non-power-of-two texture or not mipmap complete
INVALID_FRAMEBUFFER_OPERATION: clear: Attachment type must be compatible with attachment object
)

It's in __renderMultiSample, if that helps? 
```
render [Line: 12561, Col: 9], ngl.full.min.js
renderBufferDirect [Line: 12287, Col: 110], ngl.full.min.js
s [Line: 11647, Col: 17], ngl.full.min.js
render [Line: 12387, Col: 137], ngl.full.min.js
NGL.Viewer.prototype.__renderMultiSample [Line: 24802, Col: 13], ngl.full.min.js
NGL.Viewer.prototype.render [Line: 24813, Col: 264], ngl.full.min.js
NGL.Viewer.prototype.animate [Line: 24716, Col: 13], ngl.full.min.js
```

I've tried on two machines running IE11 on Windows 7 (one hadn't seen NGL before so don't think it's due to previous settings hanging around in localStorage or old versions in the cache).

Both the demo page and my local copy work fine under chrome and firefox.

**Comment by fredludlow** [comment]
Playing with git bisect I think this first appeared between arose@b27266a (working) and arose@5c461ca (not) (the imbetween commits had other minor issues that tripped up IE) but we've rapidly gone past my ability to debug stuff by this point :)

**Comment by arose** [comment]
Thanks so much for finding this! I wasn't handling the case when `OES_texture_half_float` is not available. Fixed in 927e64105ad45bbab5a7116216a751fc056c8070.

**Comment by fredludlow** [comment]
That's working again in IE11 - thanks for fixing so quickly!
